domain/src/main/java/com/zebrunner/reporting/domain/db/TestRunArtifact.java

```diff
@@ -0,0 +1,25 @@
+package com.zebrunner.reporting.domain.db;
+
+import lombok.Getter;
+import lombok.Setter;
+
+import java.io.Serializable;
+import java.util.Date;
+
+@Getter
+@Setter
+public class TestRunArtifact implements Serializable {
+
+    private static final long serialVersionUID = 4940282315917013242L;
+
+    private Long id;
+    private String name;
+    private String link;
+    private Date expiresAt;
```

Comment:
let's use LocalDateTime for new types. I misunderstood your question, thought we were talking about existing types